includes/wikia/services/TemplateClassificationService.class.php

```diff
@@ -110,6 +110,26 @@ public function classifyTemplate( $wikiId, $pageId, $templateType, $provider, $o
 	}
 
 	/**
+	 * Get all classified template types on given wiki with their page id as a key
+	 *
+	 * @param int $wikiId
+	 * @return array
+	 * @throws Exception
+	 * @throws \Swagger\Client\ApiException
+	 */
+	public function getTypesOnWiki( $wikiId ) {
```

Comment:
I'd still go with the word `Templates` in the name since ultimately what you get is a list of templates (keys are `page_id`s) with some details and not a list of types. So - `getTemplatesOnWiki`? :)